prowler/lib/outputs/finding.py

```diff
@@ -35,6 +35,7 @@ class Finding(BaseModel):
     status_extended: str
     muted: bool = False
     resource_uid: str
+    resource_metadata: dict = Field(default_factory=dict)
```

Comment:
I'd leave this commented until we have all the checks completed, not to generate inconsistent outputs, some with the `resource_metadata` and some without.